deployment/docker-compose/docker-compose.yml

```diff
@@ -18,7 +18,18 @@ services:
       watch:
         - action: rebuild
           path: ../../backend
-
-
+  chainlit-app:
+    build:
+      context: ../../chainlit
+      dockerfile: Dockerfile.chainlit
+    ports:
+      - "8100:8100"
+    volumes:
+      - .:/app  # Mount your project folder
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
+      - API_URL=http://backend:8000
```

Comment:
Should be in an `env_file` like with other containers so we don't have to edit the docker-compose file to change it